docs/tutorials/iers.rst

```diff
@@ -0,0 +1,135 @@
+:orphan:
+
+.. doctest-skip-all
+
+**********************************
+Using and updating IERS Bulletin A
```

Comment:
@bmorris3 - based on our slack conversation, you might consider adding a section here about how to silence all the warnings we were talking about.  As we concluded there, not critical now, but it seems like a logical place.